src/components/draw/partials/draw.html

```diff
@@ -53,6 +55,7 @@
             ng-if="options.showExport"
             ng-disabled="!canExport()"
             ng-click="exportKml($event)"
+            type="button"

```

Comment:
I think so, otherwise, when the draw directive is included in the feedback form, a submit is issued.

I verify again.